Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -6181,6 +6182,32 @@ def com_google_fonts_check_colorfont_tables(ttFont):
     else:
         yield PASS, "Looks good!"
 
+@check(
+    id = "com.google.fonts/check/space_on_gid1_for_colrv0",
+    rationale = """
+        A shaping bug in Windows 10 inserts whichever glyph is on GID 1
+        on top of color glyphs, spaces, and also seemingly at random,
+        for COLRv0 fonts.
+
+        Having the space glyph on GID 1 is a practical workaround for that.
+
+        See https://github.com/googlefonts/gftools/issues/609
+    """,
+    proposal = 'https://github.com/googlefonts/fontbakery/issues/3886'
+)
+def com_google_fonts_check_space_on_gid1_for_colrv0(ttFont):
+    """Put space glyph on GID 1 right after the .notdef glyph for COLRv0 fonts."""
+    SUGGESTED_FIX = ("To fix this, please reorder the glyph order so that"
+                     " the space glyph is on GID 1 right after the .notdef glyph.")
+    if 'COLR' in ttFont.keys() and ttFont['glyf'].glyphOrder[1] != 'space':
```

Comment:
Glyph may also be named `uni0020`. You can look in the cmap to get the space glyph's name.

Reply:
Why are there several cmap sub tables?

Reply:
that's besides the point, you don't need to check cmap just glyf or CFF to check if the second glyph has contours, if it does, complain loudly becuase win10

Reply:
`glyf` is for TT and `CFF ` is for OT?

Reply:
yes. You can easily check if a glyph has no contours for the glyf table by checking the glyph's `numberOfContours == 0`. For "CFF " I don't remember off the top of my head

Reply:
empty CFF charstrings only contain "endchar", for CFF2 the latter operator is ommitted and are empty empty.

Reply:
it's fine. actually you can use the AreaPen for both TrueType and CFF flavored glyphs and it will spare you the `if` statement.